addons/crm_iap_lead/views/mail_templates.xml

```diff
@@ -31,7 +31,7 @@
                     </thead>
                     <tbody>
                         <t t-foreach="people_data" t-as="people">
-                            <tr t-att-style="people_odd and 'background-color:#eeeeee' or None">
+                            <tr t-att-style="people_index % 2 and 'background-color:#eeeeee' or None">
```

Comment:
This change is not needed in this commit but it shows good practice. Since there is a desire to remove _odd, _event and _parity.